Resources/Locale/en-US/traits/traits.ftl

```diff
@@ -55,3 +55,7 @@ trait-name-Foreigner = Foreigner
 trait-description-Foreigner =
     For one reason or another you do not speak this station's primary language.
     Instead, you have a translator issued to you that only you can use.
+
+trait-name-Voracious = Voracious
+trait-description-Voracious = Nothing gets between you and your food.
+                              You eat food and drink drinks twice as fast.
```

Comment:
```suggestion
trait-description-Voracious =
	Nothing gets between you and your food.
    You eat food and drink drinks twice as fast.
```

Reply:
mix of tabs and spaces :trollface: 

![image](https://github.com/user-attachments/assets/b2d575fe-d305-49a3-836a-4b6fa4d7d324)

Reply:
Resolved with [d532057](https://github.com/Simple-Station/Einstein-Engines/pull/668/commits/d5320575d91f6fe8f2116d5cfbc7a63847cff50b)